llvm/lib/CAS/OnDiskCAS.cpp

```diff
@@ -325,6 +325,7 @@ class InternalRefVector {
     FullRefs.reserve(SmallRefs.size() + 1);
     for (InternalRef4B Small : SmallRefs)
       FullRefs.push_back(Small);
+    FullRefs.push_back(Ref);
```

Comment:
I merged this change, so should probably remove it here to avoid accidentally duplicating it.

Reply:
I thought I cleaned this up and all the some of the comments below but apparently I edited the headers in a different directory. Will fix them.